src/components/Dashboard/Form/DocumentForm.js

```diff
@@ -62,22 +64,17 @@ export default class DocumentForm extends Component {
         validate={this.validate}
         onSubmit={this.handleSubmit}
       >
-        {({ handleSubmit, submitting, values }) => (
+        {({ handleSubmit,submitting, values }) => (
           <form className={styles.Form} onSubmit={handleSubmit}>
             <FormHeader page={this.state.page} />
             {activePage}
-            <div className="buttons">
-              {page > 0 && (
-                <button type="button" onClick={this.previous}>
-                  Wróć
-                </button>
-              )}
+            <div className={styles.buttons}>
+              {page > 0 && <Button  onClick={this.previous} label="wróć"/>}
```

Comment:
Please remove additional space between Button and onClick and add a space before closing component ` />`